test/design/place.jl

```diff
@@ -0,0 +1,138 @@
+# example 1
```

Comment:
do you think we should cite the paper, here, too, just to report where the examples are borrowed from? after all, these are somehow benchmark examples.